test/e2e/apimachinery/aggregator.go

```diff
@@ -536,6 +543,132 @@ func TestSampleAPIServer(f *framework.Framework, aggrclient *aggregatorclient.Cl
 	}
 	framework.ExpectEqual(locatedWardle, true, "Unable to find v1alpha1.wardle.example.com in APIServiceList")
 
+	// As the APIService doesn't have any labels currently set we need to
+	// set one so that a watch can track events based on the following label.
```

Comment:
instead of patching the apiservice to add the label here, can we do it where the apiservice was first [created](https://github.com/kubernetes/kubernetes/blob/56d2c87ff2aefcf11337f1a1ebd76df6debc7f8a/test/e2e/apimachinery/aggregator.go#L352-L366)?

cc @p0lyn0mial

Reply:
@smarterclayton Should the issue with patching an API Service be fixed first or is it okay to change the initial setup of the API Service to include the label as suggested above? TIA